app/models/dynamic_field.rb

```diff
@@ -0,0 +1,71 @@
+class DynamicField < ActiveRecord::Base
+  include DynamicFieldStructure::Sortable
+  include DynamicFieldStructure::StringKey
+
+  module Type
+    STRING = 'string'
+    TEXTAREA = 'textarea'
+    INTEGER = 'integer'
+    BOOLEAN = 'boolean'
+    SELECT = 'select'
+    DATE = 'date'
+    CONTROLLED_TERM = 'controlled_term'
+  end
+
+  TYPES_TO_LABELS = {
+    DynamicField::Type::STRING => 'String',
```

Comment:
It might be possible to just use `STRING => 'String'` here rather than the full `DynamicField::Type::STRING`, but I'm not 100% sure.  By the way, I realize that this may be carry-over code from Hyacinth 2, so this and other comments to come may be me proposing a revision to something that I originally wrote.

Reply:
This is carry over code. We can probably remove the `DynamicField::`. Will do.

Reply:
I just realized that we can probably remove `TYPES_TO_LABELS` all together, because that's used by the front-end code and we are no longer going to be able to use it. Also, it's easy enough to convert the strings to their label counterparts with code.